Hardware/SMBIOS.cs

```diff
@@ -332,16 +334,18 @@ public class SystemInformation : Structure {
       private readonly string version;
       private readonly string serialNumber;
       private readonly string family;
+      private readonly SystemWakeUp wakeUp;
 
       public SystemInformation(string manufacturerName, string productName, 
-        string version, string serialNumber, string family) 
+        string version, string serialNumber, string family, byte wakeUp = (byte)SystemWakeUp.Unknown) 

```

Comment:
I was more thinking of using SystemWakeUp here as the parameter type instead of byte, as it doesn't seem to be used as a byte anywhere.